packages/composite-checkout/src/components/checkout-step.js

```diff
@@ -59,7 +59,7 @@ export default function CheckoutStep( {
 CheckoutStep.propTypes = {
 	className: PropTypes.string,
 	stepNumber: PropTypes.number.isRequired,
-	title: PropTypes.string.isRequired,
+	title: PropTypes.node.isRequired,
```

Comment:
I think our test is failing because of this change I made here. @sirbrillig I'm not too familiar with how the testing framework works, can you share any guidance about how I can update the test?

Reply:
No problem. It's not because of this change, actually, it's because the test is making sure there's only one place on the checkout page where the product is shown (the review step). This PR adds a second place where the product is shown, so we need to update the tests. Instead of `.toHaveLength( 1 )` they need to read `.toHaveLength( 2 )`.

Reply:
https://github.com/Automattic/wp-calypso/pull/37609/files#diff-8c8bea6a42ea110f320aead030dd0e82R84

This seems to have fixed the test. How does it look @sirbrillig ?